### PR TITLE
fix(price-feeder): Parse multiple candles from OsmosisV2 response

### PR DESCRIPTION
**Pull request by zarazan**
## Description

- Fixes only old candle prices in the price-feeder for the OsmosisV2 API

closes: [#87](https://github.com/umee-network/osmosis-api/issues/87)

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: price-feeder/CHANGELOG.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ This was released as a part of [Umee Prop 27.](https://www.mintscan.io/umee/prop
 ### Bugs
 
 - [1428](https://github.com/umee-network/umee/pull/1428) Update umeed version to an actual tag.
+- [1615](https://github.com/umee-network/umee/pull/1615) Parse multiple candles from OsmosisV2 response
```
Comment (adamewozniak) [inline price-feeder/CHANGELOG.md:62]: This needs to go under the `## [Unreleased]` section - likely will be a part of v2.0.1

**Review by adamewozniak — APPROVED** [review]
looks good! just one changelog nit

**Comment by adamewozniak** [comment]
also @zarazan please remember to put in a backport label for api-compatible price-feeder fixes 🙏

File: price-feeder/CHANGELOG.md (hunk 1 above)
Comment (zarazan) [reply price-feeder/CHANGELOG.md:62]: Whoops, thought it was. Good catch.

Comment (zarazan) [reply price-feeder/CHANGELOG.md]: https://github.com/umee-network/umee/pull/1615/commits/ee027163f9df27eb8a5b6e137e4261764885564f

**Pull request merged**